api/v1/stats/publications/StatsPublicationHandler.php

```diff
@@ -20,4 +20,29 @@ class StatsPublicationHandler extends \PKP\API\v1\stats\publications\PKPStatsPub
 {
     /** @var string The name of the section ids query param for this application */
     public $sectionIdsQueryParam = 'sectionIds';
+
+    /**
+     * @copydoc PKPStatsPublicationHandler::getAppSpecificParams()
```

Comment:
You don't need the `@copydoc` when extending a parent method. You only need it to copy over documentation from a method when that method is not inherited.